### PR TITLE
Display less than a cent fiat amounts

### LINKED ISSUE #1023: Show tenths of cents on invoice confirmation page
### Description

When entering an invoice for Yalls.org, the cost of an article is $0.006, but the App rounds it to $0.01. Would be useful to show tenths of a cent for very small invoices (e.g. < $0.05)

### Steps to reproduce the behavior

Create an invoice to view an article at Yalls.org (current cost is 150 sats, or $0.006)

### Expected behavior

Show the invoice value of $0.006.

### Actual behavior

Shows the invoice value of $0.01

### PR DESCRIPTION
**Pull request by tanx**
Closes #1023

### CODE REVIEW

**Comment by valentinewallace** [comment]
I like this concept! I propose showing anything less than $0.01 as `< $0.01` for accuracy?

**Comment by tanx** [comment]
> I like this concept! I propose showing anything less than $0.01 as `< $0.01` for accuracy?

@valentinewallace that's what the PR does already ...

File: test/unit/helper.spec.js [hunk 1]
```diff
@@ -60,6 +60,48 @@ describe('Helpers Unit Tests', () => {
...
+      expect(num, 'to match', /\${1}/);
+      expect(num, 'to match', /^< /);
+    });
+
+    it('should display less than a cent for small numbers', () => {
+      const num = helpers.formatFiat(0.0000001, 'usd');
+      expect(num, 'to match', /0[,.]0{1}1{1}/);
+      expect(num, 'to match', /\${1}/);
+      expect(num, 'to match', /^< /);
+    });
```
Comment (tanx) [inline test/unit/helper.spec.js]: @valentinewallace see this test case.

Comment (valentinewallace) [reply test/unit/helper.spec.js]: I meant it shouldn't round up to 1 cent: https://github.com/lightninglabs/lightning-app/pull/1049/files/34a065e95032d60631495c4630f7df5373b3d9f9#diff-ef34ef96c6b5f7c6aaa62b0f4b1ab4fdR71

**Pull request merged**